include/error.h

```diff
@@ -53,4 +53,8 @@
 
 #define IS_ERR_VALUE(x)	((x) < 0)
 
+//Where to put this and at_parser errors?
+//OR EWOULDREAD
```

Comment:
I think you meant EWOULDBLOCK not EWOULDREAD.
EAGAIN is shorter so let's go with it.
Regarding the question, I think this is the best spot for the errors codes to be defined.

Reply:
Yes I meant EWOULDBLOCK

Reply:
I vote for EAGAIN too. However, I would define it similar as on Linux (11).

Reply:
done